### PR TITLE
[WIP] Add institution name and full name

### PR DESCRIPTION
**Pull request by aeschylus**

### CODE REVIEW

File: spec/features/site_spec.rb [hunk 1]
```diff
@@ -24,5 +24,33 @@
...
+        click_on 'Update Site'
+
+        visit sufia.static_path action: 'agreement'
+        expect(page).to have_content('Custom Inst Name (Custom Inst Name) requires')
+      end
+    end
+
+    describe 'institution name full' do
+      before do
+        Site.update(application_name_full: 'fullname')
```
Comment (cbeer) [inline spec/features/site_spec.rb]: Typo/copy-pasta?

File: config/routes.rb [hunk 2]
```diff
@@ -16,8 +16,6 @@
   devise_for :users
   Hydra::BatchEdit.add_routes(self)
 
-  curation_concerns_collections
```
Comment (cbeer) [inline config/routes.rb]: Where did these routes go?

File: spec/controllers/sites_controller_spec.rb [hunk 3]
```diff
@@ -27,7 +27,10 @@
   # Site. As you add validations to Site, be sure to
   # adjust the attributes here as well.
   let(:valid_attributes) do
-    { application_name: "Custom Name" }
+    {
+      application_name: "Custom Name",
+      institution_name: "Custom Inst Name"
```
Comment (cbeer) [inline spec/controllers/sites_controller_spec.rb]: It seems like we should also test `institution_name_full` here, yeah?

File: db/migrate/20160412223500_add_institution_name_full_to_site.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddInstitutionNameFullToSite < ActiveRecord::Migration
+  def change
+    add_column :sites, :institution_name_full, :string
```
Comment (cbeer) [inline db/migrate/20160412223500_add_institution_name_full_to_site.rb]: Before merging, we'll probably want to collapse these two migrations for our future sanity.

File: spec/features/site_spec.rb [hunk 5]
```diff
@@ -24,5 +24,37 @@
...
+          institution_name_full: 'fullname'
+        )
+      end
+      it 'updates the institution name in the agreement text' do
+        visit edit_site_path
+        fill_in 'Institution name', with: 'Custom Inst Name'
+        click_on 'Update Site'
+
+        visit sufia.static_path action: 'agreement'
+        expect(page).to have_content('fullname (Custom Inst Name) requires')
```
Comment (cbeer) [inline spec/features/site_spec.rb]: Aren't these expectations backwards? I think the agreement text reads `institution_name (instution_name_full)`?

File: config/routes.rb (hunk 2 above)
Comment (aeschylus) [inline config/routes.rb]: Not Sure. They may have gotten lost when merging from master. I can reinstate them.

File: db/migrate/20160412223500_add_institution_name_full_to_site.rb (hunk 4 above)
Comment (aeschylus) [inline db/migrate/20160412223500_add_institution_name_full_to_site.rb]: Okay. Would I just edit the text of the migration to combine the two `add_column` statements?

File: spec/features/site_spec.rb (hunk 5 above)
Comment (aeschylus) [inline spec/features/site_spec.rb]: That's what I assumed going in, but the parenthetical seems to be the abbreviation, which also makes sense in its own way. The test also bears that out ("at least on my machine!").

Comment (aeschylus) [inline spec/features/site_spec.rb]: They seem to fail because the form submission doesn't work in the test. Only the `before` update values ever show up.

**Comment by mjgiarlo** [comment]
Merged in #77. Thanks, @aeschylus!

**Pull request closed without merging**